test/dockerfiles/Dockerfile.universal

```diff
@@ -20,7 +20,7 @@ RUN apk add --no-cache openssh curl \
   && ssh-keygen -A \
   && sed -i s/#PermitRootLogin.*/PermitRootLogin\ yes/ /etc/ssh/sshd_config \
   && sed -i s/#PermitEmptyPasswords.*/PermitEmptyPasswords\ yes/ /etc/ssh/sshd_config \
-  && echo "root:" | chpasswd
+  && echo "root:*" | chpasswd -e
```

Comment:
Is this due to a new alpine?

Reply:
Yes

Reply:
Actually the fix was a little bit different (I discovered later on a branch with dynamic tracing)